docs/build/tools/netrunner.md

```diff
@@ -0,0 +1,180 @@
+# Network-runner
+
+The network-runner tool allows to define, start and shutdown default or custom avalanchego networks for development and testing.
+
+> **Please do not use this tool for running production nodes. We cannot take responsibility nor be able to give support in this case.**
+
+## How to install
+The network-runner repository is hosted at 
+
+```
+[https://github.com/ava-labs/avalanche-network-runner](https://github.com/ava-labs/avalanche-network-runner)
+```
+
+Opening that URL will display a README file with further details of this tool.
+
+Clone the repository with
+```
+git clone https://github.com/ava-labs/avalanche-network-runner.git
+```
+
+## Supported backends
+The tool can run nodes
+
+* locally by spawning processes running on the same machine
+* using a kubernetes network as a backend
+
+The tool shares a common interface design, located at the `<REPOSITORY_ROOT>/network` package.
+Then each backend implements its own runtime.
+
+* Local: `<REPOSITORY_ROOT>/local`
+* Kubernetes: `<REPOSITORY_ROOT>/k8s`
```

Comment:
Does `REPOSITORY_ROOT` refers to github directory? I don't see <REPOSITORY_ROOT>/local and/or <REPOSITORY_ROOT>/k8s